cpp/collections_module/algorithm/collections.hpp

```diff
@@ -0,0 +1,75 @@
+#pragma once
+
+#include <mgp.hpp>
+
+namespace Collections {
+
+constexpr std::string_view kReturnSum = "sum";
+
+constexpr std::string_view kProcedureSum = "sum";
+
+constexpr std::string_view kInputList = "input_list";
+
+constexpr std::string_view kResultSum = "sum";
+
+void Sum(mgp_list *args, mgp_graph *memgraph_graph, mgp_result *result, mgp_memory *memory);
+
+constexpr std::string_view kReturnUnion = "union";
+
+constexpr std::string_view kProcedureUnion = "union";
+
+constexpr std::string_view kArgumentsInputList1 = "input_list1";
+constexpr std::string_view kArgumentsInputList2 = "input_list2";
+
+constexpr std::string_view kResultUnion = "union";
+
+void Union(mgp_list *args, mgp_graph *memgraph_graph, mgp_result *result, mgp_memory *memory);
+
+constexpr std::string_view kReturnSort = "sorted";
+
+constexpr std::string_view kProcedureSort = "sort";
+
+constexpr std::string_view kArgumentsInputList = "input_list";
+
+constexpr std::string_view kResultSort = "sorted";
+
+void Sort(mgp_list *args, mgp_graph *memgraph_graph, mgp_result *result, mgp_memory *memory);
+
+constexpr std::string_view kReturnCS = "contains";
+
+constexpr std::string_view kProcedureCS = "contains_sorted";
+
+constexpr std::string_view kArgumentInputList = "input_list";
+constexpr std::string_view kArgumentElement = "element";
+
+constexpr std::string_view kResultCS = "contains";
+
+void ContainsSorted(mgp_list *args, mgp_graph *memgraph_graph, mgp_result *result, mgp_memory *memory);
+
+constexpr std::string_view kReturnMax = "max";
+
+constexpr std::string_view kProcedureMax = "max";
+
+constexpr std::string_view kResultMax = "max";
+
+void Max(mgp_list *args, mgp_graph *memgraph_graph, mgp_result *result, mgp_memory *memory);
+
+constexpr std::string_view kProcedureSplit = "split";
+
+constexpr std::string_view kReturnSplit = "splitted";
+
+constexpr std::string_view kArgumentDelimiter = "delimiter";
+
+constexpr std::string_view kResultSplit = "splitted";
+
+void Split(mgp_list *args, mgp_graph *memgraph_graph, mgp_result *result, mgp_memory *memory);
+
+constexpr std::string_view kReturnPairs = "pairs";
+
+constexpr std::string_view kProcedurePairs = "pairs";
+
+constexpr std::string_view kResultPairs = "pairs";
+
+void Pairs(mgp_list *args, mgp_graph *memgraph_graph, mgp_result *result, mgp_memory *memory);
```

Comment:
Can we format this differently, so first group constants together and group constants for same function. And afterwards function declarations

\* Split constants *\
constexpr ....
....


\* Max constants *\

constexpr....
...

void Max ...
\\ rest of function declarations